### PR TITLE
Set boost rate and emission schedules for graviAURA

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #563 & #562

```
brownie run issue/562/rate_emissions_graviaura

brownie run issue/563/emissions_graviaura
```

**Note:** will like to post them individually for batch testing thru the ui (posting first 562 and then 563)

### CODE REVIEW

**Review by jorijnsmit — DISMISSED** [review]
looks good, address confirmed and timestamps doublechecked.

would like @mitche50 to give final green light

**Comment by petrovska-petro** [comment]
pushed another commit to allow setting args for the emissions, since likely it will not be 5k/week linear over 5 weeks, since there'd be guard state with different amounts of emissions than when it is on full open state

File: scripts/issue/563/emissions_graviaura.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main(amount=25000000000000000000000, start_ts=1655481600, end_ts=1658505600):
```
Comment (mitche50) [inline scripts/issue/563/emissions_graviaura.py:5]: We'll need to adjust this to `amount=2000000000000000000000`, `end_ts=1656086400`

Comment (sajanrajdev) [reply scripts/issue/563/emissions_graviaura.py:5]: These parameters can be passed to the script when running it, although it won't hurt to have the default being the correct ones.

**Review by mitche50 — APPROVED** [review]
LGTM - params will be passed per Saj's comments

**Pull request merged**